core/kazoo_number_manager/src/knm_telnyx_util.erl

```diff
@@ -142,9 +135,7 @@ http_options() ->
 rep({'ok', 200=Code, _Headers, <<"{",_/binary>>=Response}) ->
     ?DEBUG_APPEND("Response:~n~p~n~p~n~s~n", [Code, _Headers, Response]),
 
-    Routines = [fun(JObj) -> maybe_remove_best_effort(?SHOULD_KEEP_BEST_EFFORT, JObj) end
-               ,fun(JObj) -> maybe_filter_rates(?SHOULD_FILTER_RATES, JObj) end
-               ],
+    Routines = [fun(JObj) -> maybe_filter_rates(?SHOULD_FILTER_RATES, JObj) end],
```

Comment:
I like using named functions when possible. Could we make this be:

```erlang
Routines = [fun maybe_filter_rates/1],
...

maybe_filter_rates(JObj) -> maybe_filter_rates(JObj, ?SHOULD_FILTER_RATES).
```

Correcting arg order in `maybe_filter_rates/2`.

Thoughts?

Reply:
Yeah think that is a nicer way of doing it 👍